packages/manager/modules/backup-agent/src/BackupAgent.context.tsx

```diff
@@ -8,6 +8,6 @@ export interface BackupAgentProviderProps {
 }
 
 export const BackupAgentContext = createContext<BackupAgentProviderProps>({
-  appName: 'Backup Agent',
+  appName: 'backup-agent',
```

Comment:
This context could have just been some initialization variables.

Context is for "reactive" state: 
1. things that changes
2. and should trigger re-renders when they change

This is more global variables for your module, not global state.